src/EnergyPlus/UnitarySystem.cc

```diff
@@ -11467,6 +11532,22 @@ namespace UnitarySystems {
                     DXCoils::SimDXCoilMultiMode(
                         state, CompName, state.dataUnitarySystems->On, FirstHVACIteration, PartLoadFrac, DehumidMode, this->m_CoolingCoilIndex, FanOpMode);
                     this->m_CompPartLoadRatio = PartLoadFrac;
+                } else if (CoilType_Num == DataHVACGlobals::CoilDX_Cooling) { // CoilCoolingDX
```

Comment:
Hmm, the new coil model isn't included in SP control. I thought I added that a while back.